lms/djangoapps/course_home_api/dates/v1/serializers.py

```diff
@@ -24,6 +24,7 @@ class DateSummarySerializer(serializers.Serializer):
     link_text = serializers.CharField()
     title = serializers.CharField()
     extra_info = serializers.CharField()
+    first_component_block_id = serializers.SerializerMethodField()
```

Comment:
Couldn't you just have this be `serializers.CharField` if the method is just a `getattr`? Or do you need the default argument? Is there a case where the property isn't defined?

Reply:
There can be Course Start, Course end, Certificate etc. dates where this field isn't in the object.